src/DonationForms/Properties/FormSettings.php

```diff
@@ -314,6 +320,7 @@ public static function fromArray(array $array): self
         ) ? $array['pdfSettings'] : [];
 
         $self->designSettingsImageUrl = $array['designSettingsImageUrl'] ?? '';
+        $self->designSettingsImageAlt = ! empty($array['designSettingsImageAlt']) ? $array['designSettingsImageAlt'] : $self->formTitle;
```

Comment:
This could be simplified to a [shorthand ternary](https://stitcher.io/blog/shorthand-comparisons-in-php#shorthand-ternary-operator) (`?:`). That is, if the array key exists but is an empty string or null (otherwise you'll get undefined index notice).

Reply:
Note that the shorthand ternary behaves differently that the [null coalescing operator](https://stitcher.io/blog/shorthand-comparisons-in-php#null-coalescing-operator) (`??`).

Reply:
Good to know next time. I seem to have had mixed the FormSettings and the HeaderTemplate conditionals. The Nullish coalescing operator in fact does work for the FormSettings. At the time of testing I did not have the proper conditional in the template & had confused the two.  c73d19901923eebe6d55405710308baf5b399846

Unless Im mistaken I couldn't find FormSetting Unit Tests but I would be happy to make them for the values & defaults. How would you feel if I started a new PR for this as I'd include tests for all the settings?

Reply:
While I often suggest moving additional work in a separate PR, in the case of a test I think it would be important to add it as a part of the PR. I see that the class does not have a test yet,so if you'd like to jump on a pairing session I'd be happy to work together on adding the test.